hls4ml/backends/oneapi/passes/convolution_templates.py

```diff
@@ -233,3 +238,165 @@ def format(self, node):
             raise RuntimeError('channels_first not supported on oneAPI')
         params['data_format'] = 'cl'
         return self.template.format(**params)
+
+
+depthconv2d_function_template = (
+    'nnet::depthwise_conv_2d_{data_format}<{input_t}, {output_t}, {config}>({input}, {output}, {w}, {b});'
+)
+
+sepconv2d_function_template = (
```

Comment:
I don't think this is ever used. The converters should have split the separable to a depthwise + conv already. I would prefer not to add it. It's mainly historic in Vivado. I was tempted to get rid of it there.